vars/utils.groovy

```diff
@@ -525,9 +528,45 @@ def pytestSuite3() {
         //runsql("DELETE FROM config_tags; DELETE FROM job; DELETE FROM config;")
         sshagent (credentials: ['bastion-ssh-key']) {                 
            // test fin builder and test fin builder conv in sequence
-           sh "pytest tests/test_fin_evaluator.py -s"
-           sh "pytest tests/test_update_golden.py -s"
+           sh "python3 -m coverage run -a -m pytest tests/test_fin_evaluator.py -s"
+           sh "python3 -m coverage run -a -m pytest tests/test_update_golden.py -s"
+           sh "coverage report -m"
+           sh "python3 -m coverage json"
+
+        }
+        try {
+           sh "wget http://localhost:${jenkin_port}/job/${job_name}/job/${branch_master}/${latest_succes_build}/artifact/${cov_file}"
+           sh "python3 tests/covscripts/coverage.py branch"
+        } catch (Exception err) {
+           currentBuild.result = 'SUCCESS'
+        }
```

Comment:
If the coverage script is part of the catch and the result is ignored, then a loss of coverage will never result in failure since it would be caught and converted to success